### DISCUSSION TITLE
small size Template "PSAppDeployToolkit_Template_v4_withoutUI.zip"?

### DISCUSSION THREAD

**Discussion by snowmountainer** [PSAppDeployToolkit/PSAppDeployToolkit#1163] [Ideas]
Hello

First of all, thank you for version 4.
Personally, we don't need a UI for deployments via MECM in over 98% of cases.

Couldn't you provide a general small size template "PSAppDeployToolkit_Template_v4_withoutUI.zip"?

**Comment by mjr4077au** [comment]
The next point release will be ~3x smaller.

![image](https://github.com/user-attachments/assets/874302df-66ac-44a6-a0f0-59e3cd0e8cec)
